plugins/splunk-on-call/src/components/Incident/IncidentListItem.tsx

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import {
+  ListItem,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  Tooltip,
+  ListItemText,
+  makeStyles,
+  IconButton,
+  Typography,
+} from '@material-ui/core';
+import DoneIcon from '@material-ui/icons/Done';
+import DoneAllIcon from '@material-ui/icons/DoneAll';
+import {
+  StatusError,
+  StatusWarning,
+  StatusOK,
+  useApi,
+  alertApiRef,
+} from '@backstage/core';
+import { formatDistanceToNowStrict } from 'date-fns';
```

Comment:
oh yep, this definitely applies
https://backstage.io/docs/architecture-decisions/adrs-adr010

Reply:
I think the use of luxon is a little less relevant here (to display the duration in words particularly).

I just saw your TODO in `plugins/jenkins/src/components/Cards/Cards.tsx` that also mentions that:

```ts
// TODO This works, but hard codes as minutes. Would prefer something smarter/relative.
const durationInMinutes = Math.round(
  Duration.fromMillis(latestRun.duration).as('minutes'),
);
```

One way to do it would be to:
- Display the time in minutes (but I find that problematic in some cases)
- Display the creation date (this is the method used by the Splunk On-Call dashboard)
<img width="1123" alt="Capture d’écran 2021-02-08 à 20 44 47" src="https://user-images.githubusercontent.com/32459935/107272840-941f3a80-6a4e-11eb-9817-bf42b07bb094.png">

- Use custom logic to display the most appropriate unit value (but it adds a few more lines of code )
I did a test and the problem is that we lose the rounding of the values. (e.g `2 hours and 58 min` will become `2 hours`)

Reply:
FINE. Challenge accepted. 🤝 

https://github.com/backstage/backstage/pull/4431